### PR TITLE
Add IC and analysis modules

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Add IC and analysis modules

In this PR, illumination correction was performed on the max projected images. Then, segmentation and feature extraction (analysis) is performed on the correct images, outputting a SQLite file per plate. Other small changes are included.

### CODE REVIEW

File: 3.cp_analysis/pipeline/analysis.cppipe [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+MeasureTexture:[module_num:13|svn_version:'Unknown'|variable_revision_number:7|show_window:False|notes:['Measure texture in both nuclei and whole images across all channels. Using default parameters.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select images to measure:A647, DAPI, GOLD
+    Select objects to measure:Nuclei
+    Enter how many gray levels to measure the texture at:256
+    Hidden:1
+    Measure whole images or objects?:Both
+    Texture scale to measure:3
+
+ExportToDatabase:[module_num:14|svn_version:'Unknown'|variable_revision_number:28|show_window:False|notes:['Export data to a SQLite file. Using deault parameters for caluclating mean measurements per image.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```
Comment (jaceybronte) [inline 3.cp_analysis/pipeline/analysis.cppipe]: Small typo: default

File: 3.cp_analysis/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# CellProfiler segmentation and feature extraction
+
+In this module, we perform single nuclei segmentation using the DAPI channel. 
+Using the nuclei masks generated, features are extracted per nuclei across each channel.
+Features include intensity, granularity, texture, radial distribution, neighbors, and area + shape.
```
Comment (jaceybronte) [inline 3.cp_analysis/README.md]: Just a clarifying question - why are area and shape listed in this way?

**Review by jaceybronte — APPROVED** [review]
Hi Jenna, I think everything looks good and was explained really well! I had one question and then noticed a typo but other than that didn't notice anything

File: 3.cp_analysis/pipeline/analysis.cppipe (hunk 1 above)
Comment (jenna-tomkinson) [reply 3.cp_analysis/pipeline/analysis.cppipe]: Great catch! I will fix that.

File: 3.cp_analysis/README.md (hunk 2 above)
Comment (jenna-tomkinson) [reply 3.cp_analysis/README.md]: Fair question! Area and shape are measured in the same module in CellProfiler, so I tried to pair them up somehow. I will change this format so it is more clear.

**Comment by jenna-tomkinson** [comment]
Thanks @jaceybronte for the review! I am merging now!

**Pull request merged**